super_calendar/__openerp__.py

```diff
@@ -1,38 +1,26 @@
 # -*- coding: utf-8 -*-
-##############################################################################
-#
-#    Copyright (c) All rights reserved:
+# Copyright
 #        (c) 2012-2015 Agile Business Group sagl (<http://www.agilebg.com>)
 #                      Lorenzo Battistini <lorenzo.battistini@agilebg.com>
 #        (c) 2012      Domsense srl (<http://www.domsense.com>)
 #        (c) 2015      Anubía, soluciones en la nube,SL (http://www.anubia.es)
 #                      Alejandro Santana <alejandrosantana@anubia.es>
 #        (c) 2015      Savoir-faire Linux <http://www.savoirfairelinux.com>)
 #                      Agathe Mollé <agathe.molle@savoirfairelinux.com>
-#
-#    This program is free software: you can redistribute it and/or modify
-#    it under the terms of the GNU Affero General Public License as
-#    published by the Free Software Foundation, either version 3 of the
-#    License, or (at your option) any later version.
-#
-#    This program is distributed in the hope that it will be useful,
-#    but WITHOUT ANY WARRANTY; without even the implied warranty of
-#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
-#    GNU Affero General Public License for more details.
-#
-#    You should have received a copy of the GNU Affero General Public License
-#    along with this program.  If not, see http://www.gnu.org/licenses
-#
-##############################################################################
+#        (c) 2016      Serpent Consulting Services Pvt. Ltd.
+#                      (http://www.serpentcs.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
 {
     'name': 'Super Calendar',
-    'version': '8.0.0.2.0',
+    'version': '9.0.1.0.0',
     'category': 'Generic Modules/Others',
     'summary': 'This module allows to create configurable calendars.',
     'author': ('Agile Business Group, '
                'Alejandro Santana, '
                'Agathe Mollé, '
-               'Odoo Community Association (OCA)'),
+               'Odoo Community Association (OCA), '
+               'Serpent Consulting Services Pvt. Ltd.'),
     'website': 'http://www.agilebg.com',
     'license': 'AGPL-3',
     'depends': [
```

Comment:
web_calendar depends on web and web depdends on base.
So remove base from depends.